web-console/src/lib/functions/pipelines/status.ts

```diff
@@ -49,35 +51,40 @@ export const getDeploymentStatusLabel = (status: PipelineStatus) => {
     .exhaustive()
 }
 
-export const isPipelineIdle = (status: PipelineStatus) => {
+/**
+ * Is the pipeline ready to process API requests related to data processing
+ */
+export const isPipelineInteractive = (status: PipelineStatus) => {
```

Comment:
I like the naming of this function :+1: The only exception is the logs endpoint, but as far as I see that is already taken care of.